storage-migration-error.asc

```diff
@@ -0,0 +1,54 @@
+:docinfo1:
+= Unexpected error during Storage Migration
+:sectanchors:
+
+There was an error copying your media to a folder supported by Android R. Before the process started, your data was backed up to AnkiWeb and can be restored.
```

Comment:
I thought we didn't mandate syncing to AnkiWeb, or in fact syncing at all before migration?